spikeextractors/extractors/neuroscopeextractors/neuroscopeextractors.py

```diff
@@ -0,0 +1,451 @@
+from spikeextractors import RecordingExtractor,SortingExtractor,MultiSortingExtractor
+from spikeextractors.extractors.bindatrecordingextractor import BinDatRecordingExtractor
+import numpy as np
+from pathlib import Path
+from spikeextractors.extraction_tools import check_get_traces_args,check_valid_unit_id,read_binary
+from bs4 import BeautifulSoup
+import os
+
+from os import listdir
+from os.path import isfile, join
+
+try:
+    import bs4
+    import lxml
+    HAVE_BS4_LXML = True
+except ImportError:
+    HAVE_BS4_LXML = False
+
+class NeuroscopeRecordingExtractor(BinDatRecordingExtractor):
+    
+    """
+    Extracts raw neural recordings from large binary .dat files in the neuroscope format.
+    
+    The recording extractor always returns channel IDs starting from 0.
+    
+    The recording data will always be returned in the shape of (num_channels,num_frames).
+
+    Parameters
+    ----------
+    folder_path : str
+        Path to the folder containing the .dat file.
```

Comment:
Can you add the `subset_channels` to the docstring? Actually I would call it `channel_ids`, which is how we call it elsewhere

Reply:
So, apparently even the BinDatRecordingExtractor doesn't actually subset by `channel_ids`, it simply reads the first `len(channel_ids)` channels of that binary format. I've adjusted the variable name to more accurately reflect this and added it to the docstring!